engine.py

```diff
@@ -634,6 +635,9 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
         pm.dockControl(panel_id, area="right", content=window, label=title)
         self.log_debug("Created panel %s" % panel_id)
 
+        # Once Maya will have settled, raise the docked panel window to the top dock tab.
+        maya.utils.executeDeferred("cmds.dockControl('%s', edit=True, r=True)" % panel_id)
```

Comment:
Unfortunately "r" is the long parameter name, but I updated the comment to ... raise (with "r=True") ...